src/PosthogAnalytics.ts

```diff
@@ -16,9 +16,10 @@ limitations under the License.
 
 import posthog, { CaptureOptions, PostHog, Properties } from "posthog-js";
 import { logger } from "matrix-js-sdk/src/logger";
+import { MatrixClient } from "matrix-js-sdk";
```

Comment:
```suggestion
import { MatrixClient } from "matrix-js-sdk/src/client";
```

Importing from `matrix-js-sdk` directly can break things IIRC

Reply:
As long as you're consistent with the import it is fine, `matrix-js-sdk` maps to `/src/` or `/lib/` depending on if release or develop repsectively